Gemfile.lock

```diff
@@ -217,7 +217,7 @@ DEPENDENCIES
   will_paginate-bootstrap
 
 RUBY VERSION
-   ruby 2.4.4p296
+   ruby 2.4.6p354
```

Comment:
I really don't feel it's a nice practice updating Ruby versions frequently, this could break the production environment. Is there a greater reason for this?

Reply:
@alexalves the update was made by @icarito in #489 and merged so I just made a PR to update discrepancies. I believe he is the maintainer for this project so I didn't worry too much, but @icarito could you please advise the reason for this update?

Reply:
also travis is back up and running!

Reply:
Hi @sashadev-sky thank you for updating the Gemfile.lock !

The reason I updated is because I would like to stay in the range of officially supported versions as much as possible - and found [here](https://hub.docker.com/_/ruby?tab=description) a list of supported tags / versions of container images to be used as a Docker base, and the closest version to the one we were using was 2.4.6 (upgrade from 2.4.4). Does Ruby change much in minor version numbers? While I agree that frequent stack changes are annoying @alaxalves, we are currently in a kind of focused push to improve Mapknitter.

I would like to clarify that rather than the maintainer, I am responsible for upkeeping the production Mapknitter.Org instance. Although I have admin privileges in case I need them for administrative tasks, I am not competent in Ruby to be able to lead the way here. @jywarren is the maintainer and main developer!

That said, I am giving extra attention to Mapknitter this period as part of a project with the goal to move hosting and help factor out the export stage to a separate container.

Reply:
> Hi @sashadev-sky thank you for updating the Gemfile.lock !

Welcome!

> Does Ruby change much in minor version numbers?  

I don't know about broadly speaking but I checked out this particular version bump, it was just internal security related updates nothing we have to update on our end

> While I agree that frequent stack changes are annoying @alaxalves, we are currently in a kind of focused push to improve Mapknitter.

Agree. Let's do it all now and if anything breaks we take a step back and fix it. We still have this luxury until the repo gets very big and it becomes more difficult to track down production failures 

> I would like to clarify that rather than the maintainer, I am responsible for upkeeping the production Mapknitter.Org instance. Although I have admin privileges in case I need them for administrative tasks, I am not competent in Ruby to be able to lead the way here. @jywarren is the maintainer and main developer!
> 
> That said, I am giving extra attention to Mapknitter this period as part of a project with the goal to move hosting and help factor out the export stage to a separate container.

Got it! Thank you. You are the "Systems Gardener" 👊love that bio!! haha

Also adding no issues working on 2.4.6 for me so far.

Thanks @icarito for the update and @alaxalves for the diligence 🎉